Makefile

```diff
@@ -1,16 +1,10 @@
-ENGINE_VERSION:=$(shell grep "engine" -A 2 codacy/requirements.lock | grep version | cut -d : -f 2 | tr -d '[:blank:]')
 .PHONY: setup_helm_repos
 setup_helm_repos:
 	helm repo add codacy-stable https://charts.codacy.com/stable
 	helm repo add codacy-unstable https://charts.codacy.com/unstable
 	helm repo add codacy-incubator https://charts.codacy.com/incubator
 	helm repo add codacy-external https://charts.codacy.com/external
 
-.PHONY: update_worker_version
```

Comment:
don't you need this for the release process?

Reply:
Not anymore. All the workflows in this chart install the stable components so we no longer need to update this. The default worker image is stable as well. :)

Reply:
This is now done on the doks makefile itself (See above)

Reply:
still desent work for the release process... we need to release with this hardcoded

Reply:
Got it. 👍